packages/dashboard/src/utils/grid.ts

```diff
@@ -3,9 +3,10 @@ import { BackendStorageType, GridClient, NetworkEnv } from "@threefold/grid_clie
 import { Profile } from "@/store";
 
 export async function getGrid(mnemonic: string) {
+  const network = window.configs.APP_NETWORK as NetworkEnv;
```

Comment:
can we use the network value [here](https://github.com/threefoldtech/tfgrid-sdk-ts/blob/development/packages/dashboard/src/portal/config.ts#L10)?